ccic/bin/process.py

```diff
@@ -315,7 +315,13 @@ def download_files(download_queue, processing_queue, retrieval_settings):
             retrieval_settings.database_path, Path(input_file.filename).name
         )
         with log.log(logger):
-            input_file, clean_up = input_file.get()
+            try:
+                input_file, clean_up = input_file.get()
+            except Exception:
+                logger.exception(
+                    "Downloading of input file '%s' failed.",
+                    input_file.filename
+                )
```

Comment:
I suggest:
```python
except Exception as e:
    logger.exception(
        "Downloading of input file '%s' failed.",
        input_file.filename,
        exc_info=e
    )
```
to no throw away any information we can have on why the download failed.

Reply:
I checked it, and it seems that adding exc_info does not add any information. The traceback is plotted even if it is omitted.